paddle/trainer/tests/sample_trainer_config_qb_rnn.conf

```diff
@@ -22,7 +22,7 @@
 default_initial_std(0.1)
```

Comment:
line 19, uniitest->unittest. please update the same mistake in `sample_trainer_config_rnn.conf` accordingly. thx